INN/umbrella-sfpublicpress
# Homepage template and styling for top story

## Changes

This pull request makes the following changes:

- Adds homepage template, less, css, etc.
- Adds mixin for the top term on top of image styling and view more arrow links
- Adds styling for the homepage top story

Desktop:
![Screen Shot 2020-01-08 at 3 17 06 PM](https://user-images.githubusercontent.com/18353636/72012661-4bcd9180-322a-11ea-9fa5-ffe0e60300b1.png)

Top term hover:
![Screen Shot 2020-01-08 at 3 17 30 PM](https://user-images.githubusercontent.com/18353636/72012660-4bcd9180-322a-11ea-8fad-5231f2dd4b10.png)

Mobile:
![Screen Shot 2020-01-08 at 3 17 46 PM](https://user-images.githubusercontent.com/18353636/72012659-4bcd9180-322a-11ea-9d42-81d385e557b8.png)

## Why



For #14

## Testing/Questions

Features that this PR affects:

- Homepage template
- Top term on top of image styling
- View more arrow links
- Homepage top story


Questions that need to be answered before merging:

- [x] Does the homepage top story look ok?
- [ ] Does the arrow link mixin look ok?
- [x] Does the top story function as expected?
- [ ] Are the colors ok for the Largo Recent Posts widget styling?

Steps to test this PR:

1. Select a post and make it the homepage top story
2. Give it a top term
3. View it on the frontend on desktop and mobile and verify it looks good
